reactor-netty5-http/src/test/java/reactor/netty5/http/client/HttpClientTest.java

```diff
@@ -1169,7 +1170,7 @@ void withConnector() {
 		disposableServer = createServer()
 		                             .handle((req, resp) ->
 		                                 resp.sendString(Mono.just(req.requestHeaders()
-		                                                              .get("test"))))
+		                                                              .get("test").toString())))
```

Comment:
This may produce `NullPointerException`

Reply:
I have added a "getHeader" helpers method in BaseHttpTest.java, which avoids to do the toString() everywhere.
There are many other places where toString() was used, so I also fixed with the getHeader helper, please check.